### DISCUSSION TITLE
When will the new schema for the Prescription drugs file be posted?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#714] [Q&A] [answered]
Now that the Departments have decided to start enforcing the requirement for the Prescription Drugs file (https://www.dol.gov/agencies/ebsa/about-ebsa/our-activities/resource-center/faqs/aca-part-61), when will the (new) schema for this file be (re-)posted?  (Wasn't the schema just up a few days ago?)

Also, how will this file interact with the "table-of-contents" file? Will there be a new field for these files in the Reporting Structure Object?

**Comment by santaviccap** [comment]
@shaselton-usds

**Comment by shaselton-usds** [comment, marked as answer]
Hey @nmayle,

Development on Github will start back up for the Rx file. As noted in the last paragraph [answer to question #1](https://www.dol.gov/agencies/ebsa/about-ebsa/our-activities/resource-center/faqs/aca-part-61), the timeline is still being determined around the requirements for disclosure. The schema itself will need to be brought up to speed with the development of the other files. The Rx schema hasn't been "up" for some time now.

The table of contents will most likely be updated to account for the additional MRF.

**Reply by DZ-DRx** [reply]
Does anyone know if a schema has been created for the pharmacy MRF?

**Reply by tdfow** [reply]
The schema page for Rx still has this:  "Consistent with the Departments’ guidance published [August 20, 2021](https://www.cms.gov/CCIIO/Resources/Fact-Sheets-and-FAQs/Downloads/FAQs-Part-49.pdf), we are not promulgating “form and manner” guidance at this time. Any material currently or previously published on this page has been superseded and is not guidance of the Departments."   So until that is updated, I would say it is still TBD

**Reply by DZ-DRx** [reply]
I've been reading various comments on the web and heard there is no schema yet and what was done is no more from 2021. But at the same time they got rid of the moratorium for enforcement on the pharmacy MRF. Seems like you would not get rid of the moratorium on enforcement until after the schema had been released.

**Reply by tdfow** [reply]
Agree, as there needs to be a schema in order for it to be created and/or used before it can be enforced.
